packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -1203,4 +1203,102 @@ void main() {
       await testColor(color);
     }
   });
+  testWidgets('RefreshIndicator default elevation value correctly',
+    (WidgetTester tester) async {
+  const List<double> positions = <double>[50.0, 100.0, 150.0];
+
+    final AnimationController positionController =
+        AnimationController(vsync: const TestVSync());
+    addTearDown(positionController.dispose);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: RefreshIndicator(
+          onRefresh: refresh,
+          child: ListView(
+            physics: const AlwaysScrollableScrollPhysics(),
+            children: const <Widget>[Text('X')],
+          ),
+        ),
+      ),
+    );
+
+    RefreshProgressIndicator getIndicator() {
```

Comment:
I would suggest avoiding defining functions as much as possible, especially in tests. The more layers a reader has to get through to figure out what's going on within a block of code, the more time is wasted. Especially for functions that don't encapsulate much, or are only used a few amount of times.